docker/templates/Dockerfile.j2

```
@@ -147,7 +147,25 @@ ADD pipeline/default.conf pipeline/logstash.conf
 RUN chown --recursive logstash:root config/ pipeline/
 # Ensure Logstash gets the correct locale by default.
 ENV LANG={{ locale }} LC_ALL={{ locale }}
-ADD env2yaml/env2yaml /usr/local/bin/
+
+# Copy over the appropriate env2yaml artifact
+COPY env2yaml/env2yaml-amd64 env2yaml/
+COPY env2yaml/env2yaml-arm64 env2yaml/
```

I don't think we need to have two layers to perform the copy:

```suggestion
COPY env2yaml/env2yaml-amd64 env2yaml/env2yaml-arm64 env2yaml/
```